routing/routing_tests/index_graph_tools.cpp

```diff
@@ -270,6 +299,29 @@ unique_ptr<SingleVehicleWorldGraph> BuildWorldGraph(unique_ptr<ZeroGeometryLoade
                                               estimator);
 }
 
+unique_ptr<TransitWorldGraph> BuildWorldGraph(unique_ptr<TestGeometryLoader> geometryLoader,
+                                              shared_ptr<EdgeEstimator> estimator,
+                                              vector<Joint> const & joints,
+                                              TransitData const & transitData)
+{
+  auto indexGraph = make_unique<IndexGraph>(move(geometryLoader), estimator);
+  indexGraph->Import(joints);
+
+  auto transitGraph = make_unique<TransitGraph>(kTestNumMwmId, estimator);
+  TransitGraph::GateEndings gateEndings;
+  MakeGateEndings(transitData.m_gates, kTestNumMwmId, *indexGraph, gateEndings);
+  transitGraph->Fill(transitData.m_stops, transitData.m_edges, transitData.m_lines, transitData.m_gates, gateEndings);
```

Comment:
Is it possible to pass transitData to Fill? Instead of passing all these fields?

Reply:
done